src/modules/renderTasks.js

```diff
@@ -0,0 +1,39 @@
+import { getTasks } from './localStorage.js';
+
+function renderTasks() {
```

Comment:
In this part of your code, I suggest you use arrow functions to follow JavaScript best practices for the ES6 approach as I mentioned before.